src/main/java/com/tekenable/config/AuditConfigurator.java

```diff
@@ -10,26 +14,24 @@
 
 public class AuditConfigurator {
```

Comment:
The entire process is just one method addAuditStuff. 
Can you refactor this into a number of methods that we can then mock out for unit testing.
List<String> getTableNames()
List<String> getColumnsForTable(tableName)
List<StringBuilder> generateAuditTableSQL(<whatever args you need>)
List<StringBuilder> generateAuditTriggerSQL(<whatever args you need>)

Concatenate the two Lists
Append a call to commit to the overall List
Call runGeneratedSQL(List<StringBuilder>) method which iterates over each StringBuilder in the List and executes the statements one after the other surrounded by a try/catch that rolls back in the event that any of the SQL statements fail.

This will allow us to unit test the class as we're not making jdbc calls intermittently in the loops.